streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDbSessionBytesStoreSupplier.java

```diff
@@ -55,8 +53,6 @@ public String metricsScope() {
 
     @Override
     public long segmentIntervalMs() {
-        return Segments.segmentInterval(
-            retentionPeriod,
-            NUM_SEGMENTS);
+        return Math.max(retentionPeriod / 2, 60_1000L);
```

Comment:
`60_1000L` -> `60_000L` ? (originally it was `MIN_SEGMENT_INTERVAL = 60 * 1000L`)

Reply:
oh! Thank you!